include/riak_kv_index.hrl

```diff
@@ -45,4 +45,30 @@
           max_results :: integer() | undefined
          }).
 
--define(KV_INDEX_Q, #riak_kv_index_v3).
+-record(colocation, {
```

Comment:
Kind of a nitpick, but I don't particularly like the term colocation. We've used the term partition/hashkey/hashgroup before. I'm really just bikeshedding here, but maybe we can open the bikeshed up in timeseries room.